admin_guide/out_of_resource_handling.adoc

```diff
@@ -116,21 +119,20 @@ To configure hard eviction thresholds, the following flag is supported:
 The node evaluates eviction thresholds per its housekeeping interval. It
```

Comment:
I did some minor rewrites so that it's specified that 10 seconds is the housekeeping interval.